.github/workflows/check-url-changes.yml

```
@@ -1,72 +1,72 @@
           
-          # Set warning flag if there are any changes
-          if [ ! -z "$DELETED_FILES" ] || [ ! -z "$RENAMED_FILES" ]; then
+          NAV_CHANGES=""
+          for f in $DELETED_NAV; do NAV_CHANGES+="$f: [DELETED]\n\n"; done
+          for f in $MODIFIED_NAV; do DIFF=$(git diff $BASE $HEAD -- "$f" | grep -vE '^\+\+\+|^---'); NAV_CHANGES+="$f:\n$DIFF\n\n"; done
+          for f in $RENAMED_NAV; do NAV_CHANGES+="$f [RENAMED]\n\n"; done
+          echo "DELETED_MD<<EOF" >> $GITHUB_ENV
+          echo "$DELETED_MD" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+          echo "RENAMED_MD<<EOF" >> $GITHUB_ENV
+          echo "$RENAMED_MD" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+          echo "NAV_CHANGES<<EOF" >> $GITHUB_ENV
+          echo -e "$NAV_CHANGES" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+          if [ -n "$DELETED_MD$RENAMED_MD$NAV_CHANGES" ]; then
             echo "warning=true" >> $GITHUB_ENV
           else
             echo "warning=false" >> $GITHUB_ENV
           fi
-          
-          # Print to console for logging
-          echo "Deleted files:"
-          echo "$DELETED_FILES"
-          echo -e "\nRenamed/Moved files:"
-          echo "$RENAMED_FILES"
-
-      - name: Post PR warning
+      - name: Post PR comment if relevant
         if: env.warning == 'true'
         uses: actions/github-script@v7
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
-            const issue_number = context.payload.pull_request.number;
-            const repo = context.repo;
-            const deletedFiles = `${process.env.DELETED_FILES}`.trim();
-            const renamedFiles = `${process.env.RENAMED_FILES}`.trim();
-            
-            let message = `🔍 **Documentation URL Checker**\n\nThis PR modifies documentation files in ways that could potentially create broken links.\n\n`;
-            
-            if (deletedFiles) {
-              message += `**Deleted files:**\n\`\`\`\n${deletedFiles}\n\`\`\`\n\n`;
-            }
-            
-            if (renamedFiles) {
-              message += `**Renamed/Moved files:**\n\`\`\`\n${renamedFiles}\n\`\`\`\n\n`;
-            }
-            
-            message += `🚨 Please review these changes carefully 🚨\n\n If not handled properly, broken links (404 errors) could appear. To maintain a smooth user experience, consider:\n- Adding redirects in the \`mkdocs.yml\` file from the old URLs to the new ones\n- Updating internal references to these files`;
+            const deletedMd = `${process.env.DELETED_MD}`.trim();
+            const renamedMd = `${process.env.RENAMED_MD}`.trim();
+            const navChanges = `${process.env.NAV_CHANGES}`.trim();
             
+            let message = "🔍 **Documentation & Navigation Checker**\n\nThis PR modifies `.md` or `.nav.yml/.nav.yaml` files.\n\n";
```

```suggestion
            let message = "🔍 **Documentation & Navigation Checker**\n\nThis PR modifies `.md` or `.nav.yml` files.\n\n";
```

There shouldn't be `.nav.yaml` files with an `a`.